From e5281e7d77c9574458013cee862c6d3fb316c50b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Graber?= <stgraber@ubuntu.com>
Date: Tue, 16 Apr 2019 01:46:34 +0100
Subject: [PATCH] lxd: Add temporary variable to disable shiftfs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Stéphane Graber <stgraber@ubuntu.com>
---
 lxd/daemon.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lxd/daemon.go b/lxd/daemon.go
index 23a8fd8f..f98ee76a 100644
--- a/lxd/daemon.go
+++ b/lxd/daemon.go
@@ -508,7 +508,7 @@ func (d *Daemon) init() error {
 		logger.Infof(" - unprivileged file capabilities: no")
 	}
 
-	if util.LoadModule("shiftfs") == nil {
+	if os.Getenv("LXD_SHIFTFS") != "false" && util.LoadModule("shiftfs") == nil {
 		d.os.Shiftfs = true
 		logger.Infof(" - shiftfs support: yes")
 	} else {
-- 
2.20.1
